bld/unit_testers/build-namelist_test.pl

```diff
@@ -637,6 +637,11 @@ sub make_config_cache {
                                      GLC_TWO_WAY_COUPLING=>"FALSE",
                                      phys=>"clm4_5",
                                    },
+     "freeliv wo fun"            =>{ options=>"-bgc bgc -envxml_dir .",
```

Comment:
- [ ] not sure what this title means?  is there free living nfix without fun?

Reply:
This actually is to make sure that you can't set free living nfix if FUN isn't on. So it's an error check for the namelist.

Reply:
I went ahead and clicked the checkbox, but that is technically against our convention of the author making sure these are checked. So uncheck the box if my answer is insufficient.